.github/workflows/nightly.yml

```diff
@@ -114,6 +80,9 @@ jobs:
       HERMES_WS_DIR: /tmp/hermes
       HERMES_TARBALL_ARTIFACTS_DIR: /tmp/hermes/hermes-runtime-darwin
       HERMES_OSXBIN_ARTIFACTS_DIR: /tmp/hermes/osx-bin
+      IOS_DEPLOYMENT_TARGET: "13.4"
+      XROS_DEPLOYMENT_TARGET: "1.0"
+      MAC_DEPLOYMENT_TARGET: "10.15"
```

Comment:
Added this to avoid reading cocoapods. That's in general better, we can find a way to read those from react-native rather than have a duplication.
However, these are fixed for the Release, so they should never change for 0.75